proto/viam/app/build/v1/build.proto

```diff
@@ -23,3 +27,51 @@ message StartBuildResponse {
   // github's ID for the build
   optional string github_id = 1;
 }
+
+message GetLogsRequest {
+  // module_id as prefix:name pair
+  string module_id = 1;
+  string job_id = 2;
+}
+
+// GetLogsReponse is a streaming endpoint that may have multiple messages that belong
+// to the same build_step if there are too many log lines to fit into a single gRPC
+// response
+message GetLogsResponse {
+  string build_step = 1;
+  repeated string log_lines = 2;
```

Comment:
I was going to use the same log message as the RDK, but given how little info that has, it is pretty pointless and just adds extra wasted bandwidth and overhead imo